book/src/getting-started/2-our-first-cxx-qt-module.md

```diff
@@ -176,7 +176,7 @@ Also, we can't move the `QObject`, which is why it is behind a Rust [`Pin`](http
 
 CXX-Qt will generate getters and setters for all properties of our `struct`.
 That's where the `number()` and `set_number()` functions used by `increment_number()` come from.
-For more details on what you can do with the `QObject` from Rust and what functions CXX-Qt will generate for you, take a look at the [`QObject` page](../concepts/generated_qobject.md).
+For more details on what you can do with the `QObject` from Rust and what functions CXX-Qt will generate for you, take a look at the [`QObject` page](../concepts/generated_qobject.md). For details on how to provide user defined methods for the `QObject`, take a look at the [`QProperty` section](../bridge/extern_rustqt.md#properties) and the [Available Flags](../bridge/extern_rustqt.md#available-flags)
```

Comment:
there are many things other than properties that can be done with qobjects so specifically mentioning them here probably isn't useful as then why not mention all the other things? Instead either link generally to the `extern_rustqt.md` or leave just the link to the QObject page as that then links to the extern RustQt page anyway along with other details for QObject that are outside the extern RustQt block

Reply:
I just put it there as it mentions that cxx always generates the getters and setters which is no longer true, so thought it best it to mention there that it's optional now. will delete

Reply:
Maybe to the end of the previous paragraph add .... 
```
See the [`QProperty` section](../bridge/extern_rustqt.md#properties) for details on user defined properties.
```

Reply:
done